src/lib.rs

```diff
@@ -209,6 +209,10 @@
 #![doc(html_logo_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo_small.png")]
 #![warn(missing_docs)]
 
+#[doc = include_str!("../README.md")]
```

Comment:
As I mentioned in #273, I think it'd be good to switch to `#![doc = include_str!("../README.md")]` and include the README verbatim at the top of the rustdoc similar to how we do with many of the other @RustCrypto crates